src/simulator/object_registry.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+import object
+
+ObjectId = int
+
+
+class ObjectRegistry:
+    def __init__(self):
+        self.__objects: dict[ObjectId, object.Object] = {}
+        self.__next_available_id = 0
+
+    def get_objects(self) -> dict[ObjectId, object.Object]:
+        """
+        Returns the dictionary self.__objects with ObjectIDs as keys and Objects as values.
```

Comment:
The doc-string is what is exposed to the outside users of a class. As such it's better to avoid referring to the class internals (e.g. self.__objects) in the doc-string, and better to keep it general.